AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling/Presentation/SwiftUI/Calling/Grid/Cell/ParticipantGridCellView.swift

```diff
@@ -65,11 +54,13 @@ struct ParticipantGridCellView: View {
         }
     }
 
-    func getRendererViewInfo() -> ParticipantRendererViewInfo? {
-        guard let remoteParticipantVideoViewId = getRemoteParticipantVideoViewId() else {
+    func getRendererViewInfo(for videoStreamId: String) -> ParticipantRendererViewInfo? {
+        guard !videoStreamId.isEmpty else {
             return nil
         }
-
+        let userId = viewModel.participantIdentifier
+        let remoteParticipantVideoViewId = RemoteParticipantVideoViewId(userIdentifier: userId,
```

Comment:
nitpick: since it's only being used once, might not need a variable for userId
```suggestion
        let remoteParticipantVideoViewId = RemoteParticipantVideoViewId(userIdentifier: viewModel.participantIdentifier,
```

Reply:
Updated!